example-code/quickstart.py

```diff
@@ -0,0 +1,125 @@
+import json
+import sys
+import time
+import requests 
+
+api_key = ""
+
+headers_json = {"Accept": "application/json"}
+headers_multipart = {"Accept": "multipart/form-data"}
+headers_urlencode = {
+    "Accept": "application/json",
+    "Content-Type": "application/x-www-form-urlencoded",
+}
+
+API_URL = "https://forge.sindri.app/api/"
```

Comment:
Please add an `API_VERSION="v1"` variable and then append it to the `API_URL` variable.

```python
API_VERSION = "v1"
API_URL = "https://forge.sindri.app/api/" + API_VERSION + "/"
```